.github/workflows/build.yml

```diff
@@ -168,12 +168,37 @@ jobs:
         uses: mxschmitt/action-tmate@v3
         if: env.RUN_TMATE
   test:
+    name: >-
+      test (${{ matrix.scenario }}) -
+      ${{ matrix.platform }}-${{ matrix.architecture }}
     needs:
       - diagnostics
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
+        architecture:
+          - amd64
+          - arm64
+        platform:
+          # Amazon Linux does not (yet?) provide a FreeIPA client package,
+          # which is required for this Ansible role.
+          # - amazonlinux2023-systemd
+          - debian10-systemd
+          - debian11-systemd
+          - debian12-systemd
+          # Debian Trixie does not currently offer an IPA client package:
+          # https://packages.debian.org/search?suite=all&searchon=names&keywords=freeipa
+          # - debian13-systemd
```

Comment:
Would you make an issue to track this? Debian Trixie _did_ have the package available but it was pulled because of an issue in the `sssd` package (which was also pulled). Please see:
https://tracker.debian.org/news/1521964/freeipa-removed-from-testing/
https://tracker.debian.org/news/1522104/sssd-removed-from-testing/
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1068063

Reply:
Please see #48 and cisagov/ansible-role-freeipa-client#66.

Reply:
One additional ask is would you please add that to one of the comments (whichever you feel is most suitable) to track it in the code as well?

Reply:
Please see commit 3ca4297.